app/src/main/java/com/ccc/nameapp/extension/ActivityExtension.kt

```diff
@@ -106,8 +106,14 @@ fun Activity.isKeyboardOpened(): Boolean {
     return heightDifference > Constant.Keyboard.MIN_HEIGHT_VISIBLE
 }
 
-fun Activity.observerKeyboardShow(onKeyboardShow: (Boolean, ViewTreeObserver.OnGlobalLayoutListener) -> Unit) {
-    getRootView().viewTreeObserver.addOnGlobalLayoutListener(object : ViewTreeObserver.OnGlobalLayoutListener {
+fun Activity.observerKeyboardShow(
+    onKeyboardShow: (
+        Boolean,
+        ViewTreeObserver.OnGlobalLayoutListener
+    ) -> Unit
+) {
+    getRootView().viewTreeObserver.addOnGlobalLayoutListener(object :
+        ViewTreeObserver.OnGlobalLayoutListener {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/08a2d406ec58d7b52d91c64120c3c4101a7e87e2/app/src/main/java/com/ccc/nameapp/extension/ActivityExtension.kt#L116"></span>Unexpected indentation (8) (should be 12)</td>
    </tr>
  </tbody>
</table>